modules/user/user.repository.js

```diff
@@ -86,7 +109,7 @@ class UserRepository {
     }
 
     async findMaxUserId() {
-        const usersArr = await this.readAllUsers();
+        const usersArr = await this.readAllUsers({}, {});
```

Comment:
this looks very strange.
If you need to add arguments by default just initialize function like 
`readAllUsers(searchParams = {}, sortParams = {}, limit, skip)`

- [ ] remove function call using `fn({}, {})`